### ISSUE TITLE
What does ‘face’ and ‘random’ mean in the project code?

### ISSUE THREAD

**Issue by juthiszoey** [egrowe/single_trial_NPGC_classify#1]
What does ‘face’ and ‘random’ mean in the project code? Do they represent two different experiment conditions?

**Comment by egrowe** [comment]
Yes, two different experimental conditions where either a face or random pattern were shown. See Shafto & Pitts 2015 for full details of the experimental paradigm.
